bonfire/bonfire.py

```diff
@@ -81,6 +89,51 @@ def current_namespace_or_error():
     return namespace
 
 
+def option_usage_wrapper(command):
```

Comment:
I'm still thinking we could capture the commands in 1 place more easily... do you need the wrapper? For example if we want to just record the command that was executed to start with, what if we just record the value of `sys.argv` when we are in the `main_with_handler` function?

For example you can see if you add this code:

```
def main_with_handler():
    print(sys.argv[1:])
    try:
        main()
    except FatalError as err:
        _error(str(err))
```

It results in this:
```
❯ bonfire process foo --set-parameter someapp/someparam=somevalue --clowd-env=myenv
['process', 'foo', '--set-parameter', 'someapp/someparam=somevalue', '--clowd-env=myenv']
```

Reply:
I added a function to bonfire, `send_telemetry`  that we can call anywhere. The parameter values get replaced with `******`